interactive_engine/executor/ir/runtime/src/process/entry.rs

```diff
@@ -51,6 +52,8 @@ pub enum EntryType {
     Intersection,
     /// Type of collection consisting of entries
     Collection,
+    /// A Null graph element entry type
+    NullGraphType,
```

Comment:
`Null` or `NullEntity` is a better name. Because in other items, you did not say VertexType, or EdgeType,

Reply:
Done.